lua/entities/gmod_wire_turret.lua

```diff
@@ -96,47 +96,50 @@ local ValidTracers = {
 	[""]                      = true
 }
 
-function ENT:SetSound(sound)
+function ENT:SetSound( sound )
+	local sound = tostring( sound or "" )
+	sound = string.Trim( sound ) -- The string cannot have whitespace
 	local check = string.find( sound, "[\"?]" ) -- Preventing client crashes
-	self.sound = check and "" or sound
+	sound = check and "" or sound
```

Comment:
self.sound = check~=nil and sound~="" and sound or nil

Reply:
https://github.com/wiremod/wire/pull/2075/commits/4b364bbd0bc9870d86ab82368f80b4e3cef058aa